themes/default/content/blog/redis-cloud-provider/index.md

```diff
@@ -0,0 +1,306 @@
+---
+title: "Pulumi and Redis Cloud: Real-Time Data for Modern Apps"
+date: 2023-10-01
+meta_desc: Learn how to use Pulumi to provision fully managed Redis Enterprise Cloud clusters and connect them to your AWS workloads.
+meta_image: redis-enterprise-cloud-pulumi.png
+authors:
+    - josh-kodroff
+tags:
+    - redis-cloud
+    - aws
+---
+
+Redis Enterprise Cloud makes apps faster by providing an in-memory database that enables teams to create a real-time data platform. In this post, you'll learn how to enable connectivity between Redis Enterprise Cloud and your workloads using Pulumi and the Redis Cloud provider.
+
+<!--more-->
+
+{{% notes type="info" %}}
+The full code for this example is available at <https://github.com/pulumi/examples/tree/master/rediscloud-aws-ts>. The full code differs slightly from the example below in that it has some parameterization that makes the code more portable, whereas in this post we show hardcoded values for simplicity.
+{{% /notes %}}
+
+## About Redis Enterprise Cloud
+
+[Redis Enterprise Cloud](https://redis.com/redis-enterprise-cloud/overview/) is a fully managed database as-a-service (DBaaS) that provides managed Redis clusters. Redis Enterprise Cloud clusters can be accessed via network peering in AWS or Google Cloud, or over the internet. The [Redis Cloud provider](https://www.pulumi.com/registry/packages/rediscloud/) allows you to manage Redis Enterprise Cloud resources in Pulumi programs.
```

Comment:
It feels like there should be something after "database-as-a-service (DBaaS)". DBaaS offering? DBaaS provider? I'm not sure which word is appropriate.

Reply:
Fixed (as soon as I push).